docs/source/python/dataset.rst

```diff
@@ -456,20 +456,160 @@ is materialized as columns when reading the data and can be used for filtering:
     dataset.to_table().to_pandas()
     dataset.to_table(filter=ds.field('year') == 2019).to_pandas()
 
+Another benefit of manually scheduling the files is that the order of the files
```

Comment:
"Scheduling" is a bit strange term to use in this context?

Reply:
Yes it is.  I changed it to "listing"